torch/_dynamo/variables/user_defined.py

```diff
@@ -828,6 +832,7 @@ def var_getattr(self, tx, name):
             subobj = self._getattr_static(name)
         except AttributeError:
             subobj = NO_SUCH_SUBOBJ
+            log.warning(f"NO_SUCH_SUBOBJ: {self.value} {name}")
```

Comment:
This doesn't seem like something we should warn on.  This is valid user code:
```
try:
   x = foo.x
except AttributeError:
   x = None
```
It would be super annoying to emit a warning for this.

Reply:
Let's pass exc_info=True too to get the full exception stack trace + exception object.